addons/mail/tests/common.py

```diff
@@ -639,9 +649,10 @@ def assertSentEmail(self, author, recipients, **values):
         if 'body' in expected:
             self.assertHtmlEqual(expected['body'], sent_mail['body'], 'Value for %s: expected %s, received %s' % ('body', expected['body'], sent_mail['body']))
         # beware to avoid list ordering differences (but Falsy values -> compare directly)
-        for val in list_check:
+        for val in email_list_check:
             if expected.get(val):
-                self.assertEqual(sorted(expected[val]), sorted(sent_mail[val]),
+                self.assertEqual(sorted(email_normalize(addr) for addr in expected[val]),
```

Comment:
Same here, would prefer not processing values to be sure of what we are testing (e.g. I have tests for formatted emails that would not test anything using that).